pandas/tests/frame/common.py

```diff
@@ -11,6 +15,35 @@
     from pandas._typing import AxisInt
 
 
+class DictWrapper(Mapping):
+    _dict: dict
```

Comment:
I think this might not be needed (same for _list)

Reply:
OK, I'll remove these lines.